zkevm-circuits/src/evm_circuit/param.rs

```diff
@@ -9,7 +9,7 @@ use halo2_proofs::{
 use std::collections::HashMap;
 
 // Step dimension
-pub(crate) const STEP_WIDTH: usize = 139;
+pub(crate) const STEP_WIDTH: usize = 140;
```

Comment:
This was required because [this](https://github.com/zemse/zkevm-circuits/blob/75c95a7bb6fc53024afc4527f9a2d68d9037fb2c/zkevm-circuits/src/evm_circuit/execution.rs#L1300) assert statement fails.